test/images/scripts/openshift.setup

```diff
@@ -60,18 +61,13 @@ docker cp $container:/usr/bin - | tar -C /usr -xf - bin/openshift bin/oc bin/oad
 # Runs a master if on the right address, otherwise runs a node
 cat > /openshift-prep <<EOF
 #!/bin/sh -ex
-if /usr/sbin/ip addr | /usr/bin/grep -i "52:54:00:9e:00:f1"; then
-    /usr/bin/hostnamectl set-hostname f1.cockpit.lan
-    # HACK: work around https://bugzilla.redhat.com/show_bug.cgi?id=1401561
-    /usr/bin/systemd-tmpfiles --create /usr/lib/tmpfiles.d/rpcbind.conf
-    /usr/bin/systemctl enable rpcbind
-    /usr/bin/systemctl start rpcbind
-    /usr/bin/systemctl start nfs-server
-    cmd="/usr/bin/openshift start --master=10.111.112.101"
-else
-    /usr/bin/hostnamectl set-hostname node-\$(/usr/bin/cat /sys/devices/virtual/dmi/id/product_uuid)
-    cmd="/usr/bin/openshift start node --kubeconfig=/openshift.local.config/node-f1.cockpit.lan/node.kubeconfig"
-fi
+/usr/bin/hostnamectl set-hostname f1.cockpit.lan
+# HACK: work around https://bugzilla.redhat.com/show_bug.cgi?id=1401561
+/usr/bin/systemd-tmpfiles --create /usr/lib/tmpfiles.d/rpcbind.conf
+/usr/bin/systemctl enable rpcbind
+/usr/bin/systemctl start rpcbind
+/usr/bin/systemctl start nfs-server
+cmd="/usr/bin/openshift start --master=10.111.112.101"
```

Comment:
AFAIK the "else" branch was for adding a secondary openshift machine for providing additional nodes. This commit would remove that possibility, as the host name is now always fixed to be "f1.cockpit.lan" and it always starts a master. I suppose this will break some of our openshift tests?